DigitalLearningSolutions.Web/Views/LearningMenu/Section/_TutorialCard.cshtml

```diff
@@ -3,22 +3,25 @@
 <div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
   <div class="nhsuk-card__content">
     <div class="nhsuk-grid-row">
-      <div class="@(Model.ShowLearnStatus ? "nhsuk-grid-column-three-quarters" : "nhsuk-grid-column-full")">
-        <h3 class="nhsuk-card__heading" id="@Model.Id-name">
+      <div class="nhsuk-grid-column-three-quarters">
+        <h3 class="nhsuk-card__heading completion-card-heading" id="@Model.Id-name">
           @Model.TutorialName
         </h3>
         <div class="nhsuk-u-margin-bottom-4">
           <partial name="Shared/_TutorialTimeSummary" model="@Model.TimeSummary"/>
         </div>
       </div>
-      @if (Model.ShowLearnStatus)
-      {
-        <div class="nhsuk-grid-column-one-quarter">
-          <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color">
+      <div class="nhsuk-grid-column-one-quarter">
+        <strong class="status-tag nhsuk-tag--orange float-right-additional-information">
+          Recommended
+        </strong>
+        @if (Model.ShowLearnStatus)
+        {
+          <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color float-right-additional-information">
```

Comment:
Why do we need the float right styling here if we didn't need to for the completion summary card? 🙂

Reply:
The reason is because the word "Recommended" is slightly too long and it means that during a resize, the tag overflows over the edge of the card 5-10 pixels before the responsive styling thing kicks in to put it on a new line

Reply:
Sorry, I'm not sure I understand how floating right fixes "Recommended" being too long? Based on your other comment I was assuming you were using float right since we've now got two fields on the right and it looks better if they're aligned with the right edge rather than the start of the column? 🙂

Reply:
Yeah I wasn't too sure either. I initially did the float right thing to fix alignment, then whilst testing out some non-float right solutions (as I realised that using float right meant I'd need to edit all the other cards and also cause complications with the completion summary card) I noticed the problem with "Recommended" being too long only happened during the non-float right solutions and float right somehow fixed it